zulipterminal/helper.py

```diff
@@ -442,7 +442,7 @@ def match_stream(stream: Any, text: str) -> bool:
     return False
 
 
-def match_groups(group_name: str, text: str) -> bool:
+def match_group(group_name: str, text: str) -> bool:
```

Comment:
I can see why this minimal rename could be useful, but you can explain why in the commit message. This helps ensure that reviewers and those who look at commits later understand your motivation - did you just change the name to add a commit? ;)

Reply:
It was really bothering me that all other `match_` methods were singular, but `match_groups` had an `s` at the end. :stuck_out_tongue_closed_eyes: